config/creatrs/imp.cfg

```diff
@@ -61,241 +61,308 @@ Defence = 7
-; tasks. Linked to from 'Graphics' field of instances in creature.cfg. Numbers from sprites-32
+; Graphics sprites are animations made from images, shown while performing various tasks.
+; Linked to from 'Graphics' field of instances in creature.cfg. Numbers from sprites-32.
+; The animation used for standing still.
 Stand = 556
-;Walking
+; The animation used for walking.
 Ambulate = 554
-;Dragging an object, only used by workers.
+; The animation used for dragging an object, only used by workers.
 Drag = 566
+; The animation used for attacking.
 Attack = 558
-;The animation used for digging and tunneling, but also for pissing on corpses and roaring when psycho
+; The animation used for digging and tunneling, but also for pissing on corpses and roaring when psycho.
 Dig = 558
-;Imps can smoke to heal up, other units do not. It uses the smoke and relax animations.
+; Imps can smoke to heal up, other units do not. It uses the 'Smoke' and 'Relax' animations.
 Smoke = 568
+; The animation used for relaxing.
 Relax = 562
-;Claiming floors and rooms, reinforcing walls.
+; The animation used for claiming floors and rooms, reinforcing walls.
 PrettyDance = 564
+; The animation used for when the creature gets hit.
 GotHit = 560
-;Dangling in the Hand of Evil.
+; The animation used for dangling in the Hand of Evil.
 PowerGrab = 574
-;Reaction to being slapped. Some creatures have animations.
+; Reaction to being slapped. Some creatures have animations.
 GotSlapped = 576
+; Reaction upon researching a new spell/room, manufacturing a door/trap, gaining a level or receiving gold.
 Celebrate = 564
+; The animation used for when the creature sleeps in its lair.
 Sleep = 556
+; The animation used for when the creature eats.
 EatChicken = 556
-;Being tortured. 48 is generic.
+; The animation used for being tortured. 48 is generic.
 Torture = 56
-;Used when moaning, when assigned to a job it cannot do or in prison.
+; The animation used for moaning, when assigned to a job it cannot do or in prison.
 Scream = 570
-;Dying/unconscious. Last frame is dead.
+; The animation used for dying/unconscious. Last frame is the dead body.
 DropDead = 572
-;Gib animation (from meteor/freeze). All units use 946.
+; Gib animation (from meteor/freeze). All creatures use 946.
 DeadSplat = 946
-;Unused and unknown.
+; Unused and unknown.
 GFX18 = 10
-;Creature portrait. So an image, not a sprite/animation. Numbers from gui1
+; Creature portrait. So an image, not a sprite/animation. Numbers from gui1.
 QuerySymbol = 154
-;Creature icon. So an image, not a sprite/animation. Numbers from gui2
+; Creature icon. So an image, not a sprite/animation. Numbers from gui2.
 HandSymbol = 222
-;Unused, seems to be alternate portrait list.
+; Unused, seems to be alternate portrait list.
 GFX21 = 10
 
 [sounds]
-;The first value is the index; the second value is the count
+; The first value is the index and the second value is the count.
```

Comment:
count could be explained bit clearer, it randomly selects a sound starting at idx up to idx+count

Reply:
right so:
> ; The first value is the index and the second value is the number of variant.
; Variant means it randomly selects a sound starting at index up to index+variant. Set it to 1 for a single sound.
?

Reply:
right so:
> ; The first value is the index and the second value is the number of variant.
; Variant means it randomly selects a sound starting at index up to index+variant. Set it to 1 for a single sound.

?